.env

```
@@ -12,3 +12,20 @@
+# Socket.IO Configuration
+SOCKET_PORT=3002
+ALLOWED_ORIGINS=http://localhost:3000,http://localhost:3001,http://localhost:3002,http://localhost:3003
+NEXT_PUBLIC_SOCKET_URL=http://localhost:3002
+
+# Worker Configuration
+METRICS_WORKER_CONCURRENCY=5
+
+# Redis Configuration
+REDIS_HOST=localhost
+REDIS_PORT=6379
+REDIS_DB=0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REDIS_DB key should go before the REDIS_HOST key

(UnorderedKey)